src/blueapi/service/interface.py

```diff
@@ -162,7 +160,7 @@ def get_worker_state() -> WorkerState:
     return worker().state
 
 
-def pause_worker(defer: bool | None) -> None:
+def pause_worker(defer: bool) -> None:
```

Comment:
Could: Given the previous discussion it might be nice to quickly to this and the big schema change it necessitates in a separate PR